app/controllers/api/events_controller.rb

```diff
@@ -1,4 +1,15 @@
 class API::EventsController < API::RestfulController
+  include DashboardHelper
+
+  def dashboard_by_date
+    @events = page_collection dashboard_events
+    respond_with_collection
+  end
+
+  def dashboard_by_group
+    @events = grouped dashboard_events.group_by(&:organization_id)
```

Comment:
I'm starting to think that these should be the same API call.. I don't understand why you group on the server side.